### DISCUSSION TITLE
what happened to your tts on coqui's site?

### DISCUSSION THREAD

**Discussion by king-dahmanus** [coqui-ai/TTS#1951] [General]
Hey guys, so me and my buddies were trying to generating new voices but now it does that unexpected token error all the time instead of happening only some of the time like it used to. Is there a downtime? Or is there something wrong with the thing

**Comment by erogol** [comment]
Is this a constant issue for you? Can you mail the relevant content to info@coqui.ai?

**Reply by lexkoro** [reply]
Recordings longer than 10 seconds throw the `Unexpected token '<', "<html> <h"... is not valid JSON` error message

**Comment by king-dahmanus** [comment]
it was constant for me when I tried it, but my friend has managed to create a voice yesterday. I didn't try further after that
